### PR TITLE
bind 9.14.2

### PR DESCRIPTION
**Pull request by aabdnn**
Return to the stable version of BIND

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by aabdnn** [comment]
I submitted this as a suggestion from @SMillerDev but brew doesn't like the version downgrade

**Comment by SMillerDev** [comment]
@homebrew/core what's the best course of action here?

**Comment by iMichka** [comment]
Bump the `version_scheme` to 1, so that the versioning system is reset. This is useful in case of versioning mistakes.

**Comment by aabdnn** [comment]
@iMichka thanks for the suggestion about `version_scheme`. Does it have to remain in the formula permanently?

**Comment by javian** [comment]
@aabdnn yes I'm pretty sure it does since you are creating a new series of versions that needs to be tracked. If you were to remove it the update wouldn't get tracked.

File: Formula/bind.rb [hunk 1]
```diff
@@ -1,10 +1,17 @@
 class Bind < Formula
   desc "Implementation of the DNS protocols"
   homepage "https://www.isc.org/downloads/bind/"
-  url "https://ftp.isc.org/isc/bind/9.15.0/bind-9.15.0.tar.gz"
-  sha256 "ee9a18d8be7bcbb3928c9c96a71ea08ed045bfe57d3ef74f06f6177bbbb2560f"
+
+  # BIND releases with even minor version numbers (9.14.x, 9.16.x, etc) are
+  # stable. Odd-numbered minor versions are for testing, and can be unstable
+  # or buggy. They are not suitable for general deployment.
```
Comment (MikeMcQuaid) [inline Formula/bind.rb]: Can we add this to `brew audit` please @Homebrew/maintainers?

**Comment by javian** [comment]
```
19:29:35 Successfully installed bundler-2.0.1
19:29:35 1 gem installed
19:29:35 Error: 1 problem in 1 formula detected
19:29:35 bind:
19:29:35   * C: 13: col 3: `version_scheme` (line 13) should be put before `head` (line 11)
```
can you please fix this @aabdnn ?

File: Formula/bind.rb (hunk 1 above)
Comment (javian) [reply Formula/bind.rb]: Submitted a PR for this now.

**Comment by Moisan** [comment]
One last thing: could you squash your commits in one please?

**Pull request closed without merging**